debian/copyright

```diff
@@ -0,0 +1,18 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Files: *
+Copyright: 2021 Delphix
+License: Apache-2.0
+ Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Same comment as the linux-pkg version of this change: I thought that the JDK was GPLv2?

Reply:
Ah, right.. Yea, so I think it probably makes sense to make this license statement match up with the license of the JDKs, as you suggest (i.e. GPLv2).. whereas the license of the repository match up with the license of this new build logic (i.e. Apache 2.0).

Does that sound right to you? If so, are you sure GPLv2 is the correct license for these JDKs? (I don't mean to imply your wrong, just want to double check, since I don't have an knowledge of what license they carry)

Reply:
Right. The license file of the package that ends up in the package metadata should be the license associated with the contents that the package delivers.

I actually don't think it's plain GPLv2, it's GPLv2 with some weird modifications. The full text of the license is available here: https://openjdk.java.net/legal/.

Reply:
We should probably do whatever make-jpkg does here: https://github.com/delphix/make-jpkg/blob/master/lib/javase.sh#L38

Reply:
Looking at the actual `LICENSE` file in the tarballs delivered by this package, I see this:
```
The GNU General Public License (GPL)

Version 2, June 1991

Copyright (C) 1989, 1991 Free Software Foundation, Inc.
51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA

Everyone is permitted to copy and distribute verbatim copies of this license
document, but changing it is not allowed.
...
```
So, it looks like GPLv2 is the correct license to attribute here?

Reply:
Oh, at the bottom I see this:
```
"CLASSPATH" EXCEPTION TO THE GPL

Certain source files distributed by Oracle America and/or its affiliates are
subject to the following clarification and special exception to the GPL, but
only where Oracle has expressly included in the particular source file's header
the words "Oracle designates this particular file as subject to the "Classpath"
exception as provided by Oracle in the LICENSE file that accompanied this code."
```
so, yea, looks like you're correct w.r.t. them being GPLv2 + the classpath exception

I think maybe I should just copy the contents of the `LICENSE` file from those packages into this location?

Reply:
I went ahead and put the GPLv2 + CE license text in here.. I don't think the `make-jpkg` approach would work here, since none of the tarballs we're providing in this package, have a `copyright` file to copy.. it's not clear to me, where `make-jpkg` is getting it's `copyright` file from.. I don't see any in these tarballs.